# Lines starting with '#' and sections without content
# are not displayed by a call to 'details'
#
[paths]
# Paths related to this bug.
# suggested format: REPO_PATH:LINENUMBERS


[details]
# Additional details


[expected]
# The expected result


[actual]
# What happened instead

[stacktrace]
# Exception stacktrace
$ hg b details
** unknown exception encountered, details follow
** report bug details to http://mercurial.selenic.com/bts/
** or mercurial@selenic.com
** Mercurial Distributed SCM (version 1.5.2)
** Extensions loaded: b
Traceback (most recent call last):
  File "/usr/bin/hg", line 27, in <module>
    mercurial.dispatch.run()
  File "dispatch.py", line 16, in run
  File "dispatch.py", line 30, in dispatch
  File "dispatch.py", line 50, in _runcatch
  File "dispatch.py", line 470, in _dispatch
  File "dispatch.py", line 340, in runcommand
  File "dispatch.py", line 521, in _runcommand
  File "dispatch.py", line 475, in checkargs
  File "dispatch.py", line 469, in <lambda>
  File "util.py", line 401, in check
  File "/usr/lib/python2.5/site-packages/hgext/b.py", line 610, in cmd
    ui.write(bd.details(id)+'\n')
UnboundLocalError: local variable 'id' referenced before assignment

[reproduce]
# Reproduction steps


[comments]
# Comments and updates - leave your name
By: Michael
This has been resolved, and should no longer be seen.